src/core/chartOptions/Utility/timeIntervals.ts

```diff
@@ -52,17 +53,26 @@ export function getXAxisOptions(view: View, locale: string): XAxisOptions {
                     categories: view.columnNameGroups.map(cng => cng.map(n => n[locale]).join(', '))
                 };
         }
-    } else if (view.seriesType === ESeriesType.Ordinal) {
-        return getOrdinalOptions(view, locale);
     }
-    else { // Nominal
-        return {
-            type: 'category',
-            categories: view.columnNameGroups.map(cng => cng.map(n => n[locale]).join(', ')),
-            labels: {
-                autoRotation: [-45]
-            }
-        };
+    else {
+        const labels: string[] = view.columnNameGroups.map(cng => cng.map(n => n[locale]).join(', '));
```

Comment:
Can built in number parsing functions be used here instead of regex?

Reply:
Could also be a separate utility method for more robust unit testing

Reply:
Turns out using parseFloat and isNaN yield reliable results with numbers formatted in different locales. Since it's a one liner, I think separate utility method is not necessary for this.